pkg/auth/trusted.go

```diff
@@ -23,7 +25,7 @@ func NewTrustedAuthenticator() Authenticator {
 	return trustedAuth
 }
 
-func (aut *trustedAuthenticator) Authenticate(token string) (*Namespace, error) {
+func (aut *trustedAuthenticator) Authenticate(token string, ctx ...context.Context) (*Namespace, error) {
```

Comment:
By convention, the context should be the first parameter of a function (see [here](https://golang.org/pkg/context/)).
This means that the `...` vararg will also need to go away.